x-pack/plugins/alerting/server/index.ts

```diff
@@ -28,6 +28,8 @@ export type {
   AlertInstanceContext,
   AlertingApiRequestHandlerContext,
   RuleParamsAndRefs,
+  ScopedClusterClientService,
+  EleasticsearchClientService,
```

Comment:
```suggestion
  ElasticsearchClientService,
```

Reply:
Good catch! Fixed in https://github.com/elastic/kibana/pull/123935/commits/7ca58a7175653c967b2fb0f6a424f01c800c1a23.